grisera_api/activity_execution/activity_execution_service_graphdb.py

```diff
@@ -18,8 +18,13 @@ class ActivityExecutionServiceGraphDB(ActivityExecutionService):
     arrangement_service (ArrangementService): Service used to communicate with Arrangement
     """
     graph_api_service = GraphApiService()
-    activity_service = ActivityServiceGraphDB()
-    arrangement_service = ArrangementServiceGraphDB()
+
+    def __init__(self):
+        self.activity_service = None
+        self.arrangement_service = None
+        self.scenario_service = None
+        self.experiment_service = None
+        self.participation_service = None
```

Comment:
I think we should provide types of these services:
```python
self.activity_service: ActivityServiceGraphDB = None
```

Reply:
Why do you want once to have the type ActivitityService and once ActivityServiceGraphDB? I think we should be consistent. What are the pros and cons of these two solutions?

Reply:
My mistake it should be `ActivitityService`